src/reducers/entities/admin.js

```diff
@@ -1,9 +1,11 @@
 // Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
+/*eslint no-unused-vars: ["error", { "ignoreRestSiblings": true }]*/
```

Comment:
Not sure if we should promote this to be default, but it allow me to write:
```
const {[pluginId]: plugin, ...nextState} = state;
return nextState;
```

to remove the plugin. Maybe this is just being too "clever". Let me know what you think ;P

Reply:
I see it a bit confusing, you are making an assignation to discard data. I don't see it. And I don't see promoting the rule to default, I prefer false positives than false negatives.

The way that we are doing this now is something like
```
nextPosts = {...state};
Reflect.deleteProperty(nextPosts, pluginId);
```

It's way more explicit and enough compact.

Reply:
Yeah generally I agree with @jespino on this one, I'd rather be more explicit (only 1/5 though). I must say it is quite clever though :P

Reply:
Agreed! Excellent feedback: thank you :)